sirepo/package_data/static/js/sirepo.js

```diff
@@ -2160,6 +2165,20 @@ SIREPO.app.factory('requestSender', function(cookieService, errorService, userAg
         $window.open(self.formatUrl(routeName, params), '_blank');
     };
 
+    self.openSimulation = (app, route, simId) => {
+        const m = {};
+        m[route] = routeMapLocal(route, `/${route}/:simulationId`);
```

Comment:
Why does this link need to be hardwired and not gotten from the schema?

Reply:
I considered that but opted for "expediency". Since this copies the changes in #5270, could we do it there instead, just to move this forward?

Reply:
Yes.